Week-1/Blastoise/README.md

```diff
@@ -0,0 +1,43 @@
+## Problem 1- Ankit's Phone
+
+The application compares the price of two phones namely iPhone 11 and OnePlus 7T Pro. It compares the features of the two phones and gives the output in the form of JSON file named as 'Compare.json'. It also checks for the drop in price of any of the two phones and provides the notification by email. You can compare any two mobile phones of you choice by just changing the url in code.
+
+## Getting Started
+
+Clone the project on you local machine. The main file is " Web_scrapper.py " which contains the code. 
+
+## Prerequisites
+
+You need to install the modules mentioned below for running of the code(just type in these commands in bash/command prompt)
+
+```
+   pip3 install requests
+   pip3 install bs4
+   pip3 install smtplib
+   
```

Comment:
How about adding them all up in a single file called requirements.txt?
And a command like this will solve all my problems: `pip3 install -r requirements.txt`

Reply:
I am working on it.